pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -347,9 +364,9 @@ func (r *ReconcileConstraintTemplate) Reconcile(ctx context.Context, request rec
 
 	proposedCRD := &apiextensionsv1beta1.CustomResourceDefinition{}
```

Comment:
Do we need to move to `apiextensionsv1` here?

Reply:
No reason not to, IMO

Reply:
It turns out that doing this upgrade (`apiextensionsv1beta1` --> `apiextensionsv1`) actually causes a number of test breakages.  As this PR is already large, I'm going to save those for a follow-up PR.

Does that work for you? @shomron

Reply:
There are alot more places in GK that are still using`k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1` so +1 on followup.